test/kubemark/start-kubemark.sh

```diff
@@ -198,6 +198,10 @@ function create-and-upload-hollow-node-image {
     exit 1
   fi
 
+  echo "Configuring registry authentication"
+  mkdir -p "${HOME}/.docker"
+  gcloud beta auth configure-docker -q
```

Comment:
cc @kubernetes/test-infra-maintainers I wonder if this is a step we should be doing in bootstrap?

Reply:
hummm, I remember I tried it in CI and did not work

Reply:
https://github.com/kubernetes/test-infra/pull/6839

Reply:
yet mysteriously it's working in the pull-kubernetes-kubemark-e2e-gce job?

Reply:
bump gcloud? just a guess.

Reply:
oh, I figured it out, because I ran into the same issue - `gcloud` doesn't check to see if `$HOME/.docker` exists yet, and fails if it doesn't. this PR works because we do `mkdir -p` first.

Reply:
docker isn't booted in every random job so doing it early in bootstrap is probably not the best idea. /shrug